src/main/java/com/answerdigital/answerking/response/SimpleCategoryResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.answerdigital.answerking.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SimpleCategoryResponse {
+
+    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
```

Comment:
Is this required? If so, what value does this add over not having it at all?

Reply:
yes, good catch missed it after copy pasting objects